include/aos/common/tools/fs.hpp

```diff
@@ -351,13 +351,13 @@ class FS {
     }
 
     /**
-     * Reads content of the file named by fileName into the given string.
+     * Reads content of the file named by fileName into the buffer.
      *
      * @param fileName file name.
-     * @param[out] text result string.
+     * @param[out] buff destination buffer.
      * @return Error.
      */
-    static Error ReadFileToString(const String& fileName, String& text)
+    static Error ReadFile(const String& fileName, Array<uint8_t>& buff)
```

Comment:
1. `fd >= 0` is success result.
2. Return `errno` for failed io op. aos::Error properly handles `errno`.
3. Apply above for write file as well.
4. Implement similar `WriteFile` method as well.

Reply:
fixed